### ISSUE TITLE
Feat: Add admin table filters to Post Type and Taxonomies

### ISSUE THREAD

**Issue by jpSimkins** [polylang/polylang#1057]
I am aware of the admin toolbar feature to filter between: All Languages or each language

The issue with that toolbar, is that while it does filter the content, it's not really useful from a management perspective. The admin toolbar does not alter the totals for the WP statuses.

I propose having a secondary filter. This should allow you to filter any language or all languages.

This is what WPML has:
![Screenshot from 2022-06-07 11-56-53](https://user-images.githubusercontent.com/4512118/172426712-6660e279-55a1-4243-b045-7a4ad43c0fe5.png)

While I understand this is not WPML, I believe that filter is more usable than the admin toolbar filter.
- It has proper counts for each language
  - Can easily check the total against business expectations.  The current system requires loading the page for each language to get the count from the table. As more languages are added, this can get time consuming.
- It's clear which language you are in 
  - I cannot count how many times I had to point this out to others that it's in the admin topbar

As someone who is migrating from WPML to PolyLang, this is perhaps one of the largest issues with the switch. I am working on a small plugin to add this to my sites but it would be nice if this were part of PolyLang itself.

Thank you

**Comment by Chouby** [comment]
This looks like a duplicate of #133. Am I wrong?

**Comment by jpSimkins** [comment]
@Chouby Yes and no. You are correct about it being similar due to that the totals are not reflected but what I am asking for is a similar filter like this on the tables. This is because they have the filter in the topbar but it's not very obvious (most my team didn't even notice it). 

So yes, the issue is similar but this is asking for the filter to be added to the table (which would also update the totals properly then). Hope that makes sense.
